File: short_sort/short.py
def main():
    cn = int(input())
    for _ in range(cn):
        s = input()
        if s == "abc" or s == "bac" or s == "cba" or s == "acb":
            print("YES")
        else:
            print("NO")
    
    
if __name__ == "__main__":
    main()
    

File: short_sort/short.cpp
#include <iostream>
#include <string>

using namespace std;

int main()
{
    int cn;
    cin >> cn;
    for (int i = 0; i < cn; i++)
    {
        string s;
        cin >> s;
        if (s == "abc" || s == "bac" || s == "cba" || s == "acb")
        {
            cout << "YES\n";
        }

        else
        {
            cout << "NO\n";
        }
    }
}
